dataframe-jdbc/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
         alias(kover)
         alias(ktlint)
         alias(jupyter.api)
+        alias(binary.compatibility.validator)
```

Comment:
Let's disable for a while

Reply:
From other hand, it's just a check and could help to discover stuff, which should be documented!

Reply:
It doesn't hurt to add it. It will just inform you when you make a binary breaking change, after which you can call `apiDump`, commit those and the error goes away. But most importantly it makes you aware of the changes that we'll need to inform our users about for the next release.